### PR TITLE
Update submit grid model with form values example

### PR DESCRIPTION
**Pull request by AlexanderTemnov**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,20 +3,57 @@
...
 
-* [HomeController.cs](./MVC/dxSampleT590924/Controllers/HomeController.cs)
-* [Order.cs](./MVC/dxSampleT590924/Models/Order.cs) 
-* **[Index.cshtml](./MVC/dxSampleT590924/Views/Home/Index.cshtml)**
-* [Success.cshtml](./MVC/dxSampleT590924/Views/Home/Success.cshtml)
-<!-- default file list end -->
-# How to submit DataGrid model with form values
+# DevExtreme ASP.NET MVC Form - How to submit a DataGrid model with form values
 
+The DevExtreme ASP.NET MVC [Form](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.FormBuilder-1) can collect information from all hidden inputs and post it to the Controller. The [DataGrid](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.DataGridBuilder-1) does not initially have a hidden input, and the grid is placed into the form items template. If you assign model values to the grid's [dataSource](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.DataGridBuilder-1.DataSource(DevExtreme.AspNet.Mvc.JS)) option, the model values are not passed to the server.
```
Comment (arminal) [inline Readme.md]:
```suggestion
The DevExtreme ASP.NET MVC [Form](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.FormBuilder-1) can collect information from all hidden inputs and then post it to the Controller. The [DataGrid](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.DataGridBuilder-1) does not initially have a hidden input, and the grid is placed into the form items template. If you assign model values to the grid's [dataSource](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.DataGridBuilder-1.DataSource(DevExtreme.AspNet.Mvc.JS)) option, the model values are not passed to the server. 
```

File: Readme.md [hunk 2]
```diff
@@ -3,20 +3,57 @@
...
-<p>By default, the HTML form collects information from all hidden inputs and posts it to the Controller. DataGrid is not an editor and doesn't have a hidden input. As a result, model values assigned to the grid's DataSource option are not passed to the server out of the box.</p>
-<p>You can manually create hidden inputs for each row at runtime, place them onto the form and specify their name attribute as demonstrated in the "<a href="https://www.red-gate.com/simple-talk/dotnet/asp-net/model-binding-asp-net-core/">Model Binding in ASP.NET Core</a> -> Binding to Collections" help topic before it is submitted.</p>
+This example shows how to create hidden inputs for each DataGrid row at runtime and place the hidden inputs onto the form. 
 
-<br/>
+[[Run Online]](https://codecentral.devexpress.com/t590924/)
 
+![grid-model](submit-grid-model-with-form.png)
 
+Specify the "name" attribute option of the hidden input element. Then get grid data items in **clickHandler**. Save the data items to the input element in the **CreateInputElement** function.
```
Comment (arminal) [inline Readme.md]:
```suggestion
Specify the "name" attribute option of the hidden input element. Then, get grid data items in **clickHandler**. Save the data items to the input element in the **CreateInputElement** function. 
```
What do you mean by "get grid data items in clickHandler"?

**Comment by arminal** [comment]
#GrammarReview

Requested a clarification. Short doc.

**Review by AnnaKondratova — APPROVED** [review]

File: Readme.md (hunk 2 above)
Comment (AlexanderTemnov) [reply Readme.md]: @arminal I mean data items of the grid which are obtained in the clickHandler event.  "Then, get the grid data items in the clickHandler event."

Comment (arminal) [reply Readme.md]: that still doesn't make sense
do you perhaps mean "from the clickHandler event"?

Comment (AlexanderTemnov) [reply Readme.md]: @arminal  ok

Comment (AlexanderTemnov) [reply Readme.md]: I made the corresponding change.

**Pull request merged**